package.json

```diff
@@ -38,6 +37,7 @@
   },
   "homepage": "https://github.com/craftship/yith#readme",
   "dependencies": {
+    "aws-sdk": "^2.7.10",
```

Comment:
Yer my plan was to just keep an eye on it (it should actually be in dev-dependecies and locked to `2.7.10` not `^2.7.10`, currently the bundle is ~1.7mb or so.  Low risk as from this point forward I doubt they will change it, they are pretty slow on updating things to be fair from past experience.